Cannot find module './lib/fingerprint.js' from 'index.js'

**cuid:** v2.0.1
**Node.JS:** v8.9.1
**npm:** v5.5.1
**yarn:** v1.3.2
**Import style:** require
**Environment:** TypeScript
**OS:** MacOS High Sierra 10.13.1

When using `require` to load `cuid`, the error `Cannot find module './lib/fingerprint.js' from 'index.js'` is thrown at resolve time, rendering v2.0.1 unusable.

Looking in the node_module folder, it seems like the lib files are not included in the npm module. It doesn't seem to be an .npmignore problem since you don't have an .npmignore file.

Cheers!

--- a/package.json
+++ b/package.json
@@ -13,9 +13,6 @@
   "react-native": {
     "./lib/fingerprint.js": "./lib/fingerprint.react-native.js"
   },
-  "files": [
-    "dist"
-  ],
   "keywords": [
     "id",
     "guid",

